lib/sles4sap/console_redirection.pm

```diff
@@ -83,13 +183,12 @@ This changes global setting $testapi::distri->{serial_term_prompt} which is impo
 
 sub set_serial_term_prompt {
     $testapi::distri->{serial_term_prompt} = '';    # reset previous prompt first
-    my $current_user = script_output('whoami');
-    $testapi::distri->{serial_term_prompt} = ($current_user eq 'root' ? '# ' : '> ');
+    $testapi::distri->{serial_term_prompt} = script_run('whoami | grep root', quiet => 1) == 0 ? '# ' : '> ';
```

Comment:
is it `$testapi::distri->{serial_term_prompt} = '';` needed anymore?

Reply:
Yes it is. 
It turns off temporarily the prompt sign so later `script_run('whoami | grep root', quiet => 1)` won't fail.

Reply:
maybe add a comment in the code to explain it

Reply:
done